addons/bus/static/src/workers/websocket_worker.js

```diff
@@ -229,7 +251,10 @@ export class WebsocketWorker {
      * Triggered when a connection failed or failed to established.
      * Apply an exponential back off to the reconnect attempts.
      */
-    _onWebsocketError() {
+    _onWebsocketError(ev) {
+        if (this.isDebug && ev) {
+            console.debug("%c[onError]", "color: #c6e; font-weight: bold;", ev);
+        }
```

Comment:
Would have been best to share code between `_onWebsocketError` and `_onWebsocketClose` with a properly named function like `_retryConnectionWithDelay`.
That way you wouldn't need to guard `ev`